### PR TITLE
Adding Adfraud2.py file and trainsample file

### PR DESCRIPTION
**Pull request by Priyabrata409**
If you face any issue , Just let me know.I would be glad to help you out.

### CODE REVIEW

File: Adfraud2.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+Original file is located at
+    https://colab.research.google.com/drive/1UUZkSOZKVPxtHbZHcO1TQPOJpNXG9UK_
+"""
+
+import pandas as pd
+import numpy as np
+import random
+random.seed(100)
+
+data=pd.read_csv("train_sample.csv")
```
Comment (arita37) [inline Adfraud2.py:15]: replace data= with df=

File: Adfraud2.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+#########################################################################################################
+
+data.is_attributed.value_counts(normalize=True)
+
+data.drop(columns=["click_time"],axis=1,inplace=True)
+
+data["is_attributed"]=data["is_attributed"].astype("uint8")
+
+from sklearn.model_selection import train_test_split
+train_X,test_X,train_y,test_y=train_test_split(data.drop("is_attributed",axis=1),data["is_attributed"],stratify=data["is_attributed"],test_size=0.15)
```
Comment (arita37) [inline Adfraud2.py:80]: dfX =   df.drop("is_attributed",axis=1)
dfy  =  df["is_attributed"]

File: Adfraud2.py [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+data=pd.read_csv("train_sample.csv")
+data.head()
+
+data.drop(columns=["attributed_time"],inplace=True)
+data["click_time"]=pd.to_datetime(data["click_time"])
+
+"""## Let's Perform Some Feature Engineering"""
+
+## Let's Extract some features out of all availabel columns
+##########################################################################################################
```
Comment (arita37) [inline Adfraud2.py:24]: remove ########  by blank line

File: Adfraud2.py [hunk 4]
```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
```
Comment (arita37) [inline Adfraud2.py:1]: Follow the specs : 
     put train_sample.csv in  data/input/adfraud/train/feature.zip
     
     df = pd.read_csv("train/featues.zip")

File: Adfraud2.py [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+"""### Let's apply count encoding"""
+
+import category_encoders as ce
+
+
+target_enc = ce.CountEncoder(cols = cat_features)
+
+target_enc.fit(train_X[cat_features], train_y)
+
+train_encoded = train_X.join(target_enc.transform(train_X[cat_features]).add_suffix("_count"))
```
Comment (arita37) [inline Adfraud2.py:177]: train_encode_count = target_enc.transform(train_X[cat_features]).add_suffix("_count")
train_X_all                = train_X.join(    train_encode_count  )

File: Adfraud2.py [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+    'max_bin': 100,
+    'subsample': 0.9,  # Was 0.7
+    'subsample_freq': 1,
+    'colsample_bytree': 0.7,
+    'min_child_weight': 0,
+         'min_split_gain': 0,
+    'reg_alpha': 0,
+    'reg_lambda': 0,}
+num_round = 1000
+bst = lgb.train(param, dtrain, num_round, valid_sets=[dvalid], early_stopping_rounds=20)
```
Comment (arita37) [inline Adfraud2.py:245]: Can you add this one :

from gplearn.genetic import SymbolicTransformer
col_select =  [ 'col1',  'col2']
coly = 'Attributes'

def genetic_feat(df, num_gen=20, num_comp=10):
  function_set = ['add', 'sub', 'mul', 'div',
                  'sqrt', 'log', 'abs', 'neg', 'inv','tan']

  gp = SymbolicTransformer(generations=num_gen, population_size=200,
                          hall_of_fame=100, n_components=num_comp,
                          function_set=function_set,
                          parsimony_coefficient=0.0005,
                          max_samples=0.9, verbose=1,
                          random_state=0, n_jobs=6)

  gen_feats = gp.fit_transform(df[col_select], df[ coly ] )
  gen_feats = pd.DataFrame(gen_feats, columns=["gen_"+str(a) for a in range(gen_feats.shape[1])])
  gen_feats.index = df.index
  return pd.concat((df,gen_feats),axis=1)

df_out = interact.genetic_feat(df.copy()); df_out.head()

File: Adfraud2.py (hunk 1 above)
Comment (Priyabrata409) [reply Adfraud2.py:15]: Okk I will do it.

File: Adfraud2.py (hunk 5 above)
Comment (Priyabrata409) [reply Adfraud2.py:177]: Friend, We need to first fit the data, Then only we will be able to transform that
